### PR TITLE
fix: Added signing pad disable state while submitting form

### PR DESCRIPTION
**Pull request by plxity**
Fixes : #891

### CODE REVIEW

File: apps/marketing/src/components/(marketing)/widget.tsx [hunk 1]
```diff
@@ -399,6 +399,7 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
           </DialogDescription>
 
           <SignaturePad
+            containerClassName={isSubmitting ? 'pointer-events-none opacity-50' : undefined}
```
Comment (dguyen) [inline apps/marketing/src/components/(marketing)/widget.tsx:402]: Please create a `disabled` prop on the SignaturePad and update the logic there.

Comment (plxity) [reply apps/marketing/src/components/(marketing)/widget.tsx]: @dguyen I've added `isDisabled` props on the signing pad and have fixed the usage at all the places. Please review.

Comment (dguyen) [reply apps/marketing/src/components/(marketing)/widget.tsx]: Can you please use `disabled` so it matches normal input elements

File: packages/ui/primitives/signature-pad/signature-pad.tsx [hunk 2]
```diff
@@ -214,7 +216,13 @@ export const SignaturePad = ({
...
   return (
-    <div className={cn('relative block', containerClassName)}>
+    <div
+      className={cn(
+        'relative block',
+        containerClassName,
+        isDisabled ? 'pointer-events-none opacity-50' : null,
+      )}
+    >
       <canvas
```
Comment (dguyen) [inline packages/ui/primitives/signature-pad/signature-pad.tsx:226]: You can do something like this to make it neater

```
      className={cn('relative block', containerClassName, {
        'pointer-events-none opacity-50': disabled,
      })}
```

Comment (plxity) [reply packages/ui/primitives/signature-pad/signature-pad.tsx]: @dguyen Done

Comment (dguyen) [reply packages/ui/primitives/signature-pad/signature-pad.tsx]: Cheers!

Did you see the other note about using `disabled` instead of `isDisabled`?

Comment (dguyen) [reply packages/ui/primitives/signature-pad/signature-pad.tsx]: `disabled` matches how we disable inputs, buttons, fields.

`isSubmitting` is simply a variable we pass around.

Comment (plxity) [reply packages/ui/primitives/signature-pad/signature-pad.tsx]: Sure, made the changes. Please review

**Pull request merged**